tests/_support/AcceptanceTester.php

```diff
@@ -48,7 +45,7 @@ public function loginIntoJoomlaAdministratorWithUsernameAndPassword($username, $
 	public function iSeeAdministratorDashboard()
 	{
 		$I = $this;
-		$I->see('Control Panel', 'h1');
+		$I->waitForPageTitle('Control Panel', 4);
```

Comment:
This function is written in https://github.com/joomla-projects/joomla-browser/pull/115/files#diff-676cca4b1f4fdd04825e9c7ae7b54691R917 PR.